specutils/tests/test_loaders.py

```diff
@@ -121,28 +121,38 @@ def test_hst_stis(remote_data_path):
 def test_sdss_spec():
     sp_pattern = 'spec-4055-55359-0596.fits.'
     with urllib.request.urlopen('https://dr14.sdss.org/optical/spectrum/view/data/format%3Dfits/spec%3Dlite?mjd=55359&fiberid=596&plateid=4055') as response:
-        with tempfile.NamedTemporaryFile(prefix=sp_pattern) as tmp_file:
-            shutil.copyfileobj(response, tmp_file)
+        # On Windows, NamedTemporaryFile cannot be opened a second time while
+        #  already being open, so we avoid using that method.
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            file_path = os.path.join(tmp_dir, sp_pattern)
```

Comment:
I normally prefer to use the tmpdir or tmp_path pytest fixture to avoid the Windows issue